eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementInputLibrary.cs

```diff
@@ -173,5 +202,37 @@ ResourceMetadata BuildResourceMetadata(InputDecoratorInfo decorator)
                     parentResource);
             }
         }
+
+        internal bool IsResourceUpdateModel(InputModelType model)
+        {
+            if (_resourceUpdateModelToResourceNameMap.ContainsKey(model))
```

Comment:
use `TryGetValue(model, out var value)` here, so that there is 1 operation from the dictionary instead of 2.